cache/lru.go

```diff
@@ -146,6 +146,10 @@ func (c *LRUCache) Size() int {
 	return c.ll.Len()
 }
 
+func (c *LRUCache) Stats(_ bool) Stats {
+	return c.stats
```

Comment:
Not sure how this bool is intended to be used, and it seems the interface was modified in a newer version of Gubernator than ratelimits was expecting

Reply:
It was intended as a clear flag. AKA, return the current stats and reset them to zero if passed a true. I don't think it's needed anymore so this is fine.